src/store/modules/aeternity.js

```diff
@@ -63,45 +52,39 @@ export default {
     },
   },
   actions: {
-    async initTippingContractIfNeeded({
-      commit,
+    async initTippingContract({
```

Comment:
you can keep `IfNeeded` behaviour by putting contracts to a variable outside state, anyway contracts not expected to be reactive 🤔

Reply:
How expensive is it to initialize the contracts when we need them? At the moment we only need them to do stateful actions (eg. post a tip or similar). And when we have another 10ms then I don't mind creating a fresh instance.

Reply:
Not expensive at all because don't need to compile if ACI provided 👌